src/main/java/stores/SerializerStore.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * COPYRIGHT: FREQUENTIS AG. All rights reserved.
+ *            Registered with Commercial Court Vienna,
+ *            reg.no. FN 72.115b.
+ */
+package stores;
+
+import exceptions.StoreException;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import services.AddressBook;
+
+public class SerializerStore implements Store {
+
+    @Override
+    public void store(final AddressBook objectToStore) throws StoreException {
+        try (FileOutputStream out = new FileOutputStream(Store.LOCATION)) {
+            ObjectOutputStream outputStream = new ObjectOutputStream(out);
+            outputStream.writeObject(objectToStore);
+            outputStream.close();
```

Comment:
`outputStream` may not get closed in case an exception occurs while writting. It could be closed in a finally block or using try with resources like `ObjectOutputStream out = new ObjectOutputstream(new FileOutputStream(location)) {...}`